.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Should discuss, bug, needs refinement, triage
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### To Reproduce
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Expected behavior
```

Comment:
Maybe add a section with the current behavior as well

Reply:
Isn't that the "Describe the bug" section?